registration-processor/core-processor/registration-processor-abis-handler-stage/src/main/java/io/mosip/registration/processor/abis/handler/stage/AbisHandlerStage.java

```diff
@@ -681,7 +681,7 @@ private void validateBiometricRecord(BiometricRecord biometricRecord, List<Strin
 					BIR bir = optionalBIR.get();
 					Map<String, String> othersMap = new HashMap<String, String>();
 					if (bir.getOthers() != null) {
-						othersMap = bir.getOthers().stream()
+						othersMap = bir.getOthers().entrySet().stream()
```

Comment:
Can getOthers or entrySet return null?

Reply:
in line number 683 already null check is done Sasi. 
if (bir.getOthers() != null) {

so entrySet will also not throw null pointer.